source/extensions/access_loggers/http_grpc/BUILD

```diff
@@ -31,15 +31,28 @@ envoy_cc_library(
     ],
 )
 
+envoy_cc_library(
+    name = "grpc_access_log_proto_descriptors_lib",
+    srcs = ["grpc_access_log_proto_descriptors.cc"],
+    hdrs = ["grpc_access_log_proto_descriptors.h"],
+    deps = [
+        "//source/common/config:protobuf_link_hacks",
```

Comment:
I don't think the link hacks apply in this situation (see the comment in the respective file).

Reply:
Ah yep, makes sense - this is cruft left over from me copying the dependencies.